### PR TITLE
Implement "Update all" modules feature

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR adds a button near the modules to update. Ping @superGB for approval.
| Type?         | new feature
| Category?     | BO
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | /
| How to test?  | You must have several modules to update in order to test the feature.

### Preview
capture d ecran 2016-11-03 a 17 26 00

### CODE REVIEW

**Comment by superGB** [comment]
Ok for me !

File: admin-dev/themes/default/js/bundle/module/module.js [hunk 1]
```diff
@@ -44,6 +44,12 @@ var AdminModuleController = function() {
   this.categoryGridItemSelector = '.module-category-item';
   this.addonItemGridSelector = '.module-addons-item-grid';
   this.addonItemListSelector = '.module-addons-item-list';
+  
+  // Upgrade All selectors
+  this.upgradeAllSource = '.module_action_menu_upgrade_all';
+  this.upgradeAllTargets = '#modules-list-container-update .module_action_menu_upgrade';
```
Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/module/module.js]: Maybe we should add a `:visible` condition

Comment (aleeks) [inline admin-dev/themes/default/js/bundle/module/module.js]: What do you mean :visible ? They aren't all visible in this page ?

Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/module/module.js]: I've just discovered that if we spam the "Update all" button, the request is sent multiple time.
When a request is sent, the buttons are hidden. We can use their visibility to check if the previous one is finished.

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thank you @Quetzacoalt91

**Pull request merged**